docs/standard/whats-new/whats-new-in-dotnet-standard.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "What's new in the .NET Standard"
+ms.custom: "updateeachrelease"
+ms.date: "10/31/2017"
+ms.prod: ".net"
+ms.topic: "article"
+ms.technology: dotnet-standard
+ms.##devlang: dotnet
+author: "rpetrusha"
+ms.author: "ronpet"
+manager: "wpickett"
+---
+
+# What's new in the .NET Standard
+
+The .NET Standard is a formal specification that defines a versioned set of APIs which must be available on .NET implementations that comply with that version of the standard. The .NET Standard is targeted at library developers. It aims to guarantee the functionality that is available to a library that targets a version of the .NET Standard on each .NET implementation. In other words, a library that targets a version of the .NET Standard can be used on any .NET Framework, .NET Core, or Xamarin implementation that supports that version of the standard.
```

Comment:
I would change "It aims to guarantee the functionality that is available to a library that targets a version of the .NET Standard on each .NET implementation. In other words, a library that targets a version of the .NET Standard can be used on any .NET Framework, .NET Core, or Xamarin implementation that supports that version of the standard." to something like "A library that targets a .NET Standard version can be used on any .NET Framework, .NET Core, or Xamarin implementation that supports that standard version."